### PR TITLE
New id for ytdownloader

### PR DESCRIPTION
**Pull request by aandrew-me**
**I am the developer of ytdownloader - https://github.com/flathub/me.aandrew.ytdownloader
I want to change the App ID, so I am making a new pull request.**

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by aandrew-me** [comment]
bot, build io.github.aandrew_me.ytdn

**Comment by aandrew-me** [comment]
bot, build io.github.aandrew_me.ytdn

**Comment by aandrew-me** [comment]
bot build, io.github.aandrew_me.ytdn

**Comment by aandrew-me** [comment]
bot, build io.github.aandrew_me.ytdn

**Comment by aandrew-me** [comment]
bot, build io.github.aandrew_me.ytdn

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is still disappointing.

**Comment by hfiguiere** [comment]
Also you will need to EOL the original package. It also will make the transition for users harder as they will have to either manually copy the settings or recreate them.

**Comment by aandrew-me** [comment]
> Also you will need to EOL the original package. It also will make the transition for users harder as they will have to either manually copy the settings or recreate them.

Yes I will add EOL. Well nothing to do, I don't have the domain aandrew.me anymore

File: flathub.json (hunk 1 above)
Comment (aandrew-me) [reply flathub.json:2]: I don't know how to do it

Comment (hfiguiere) [reply flathub.json:2]: how do you build it?

FFMPEG is build on so many project on flathub that there is absolutely no secrets.

Comment (aandrew-me) [reply flathub.json:2]: I don't want to build it. I just need to download a different build

Comment (hfiguiere) [reply flathub.json:2]: you should be building it.

Comment (aandrew-me) [reply flathub.json:2]: Does flathub buildbot have docker?

Comment (hfiguiere) [reply flathub.json:2]: docker is not a build system.

Comment (aandrew-me) [reply flathub.json:2]: The custom ffmpeg that I am using requires docker to build

**Comment by barthalion** [comment]
/merge

Please also create an issue with repository archival request.

File: flathub.json (hunk 1 above)
Comment (aandrew-me) [reply flathub.json:2]: I think I finally figured it out. I am feeling stupid that I didn't think of that before. All I had to do was download both ffmpeg binaries, then check the processor arch and use the required binaries

**Pull request closed without merging**